aws-bom/build.gradle

```diff
@@ -1,3 +1,10 @@
+plugins {
```

Comment:
Are these the correct plugins for a `bom` submodule?

Reply:
For now this is correct, I didn't get to write a BOM plugin now, like for the module plugin.